src/main/java/com/mixpanel/android/mpmetrics/MPDbAdapter.java

```diff
@@ -102,6 +102,13 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
             db.execSQL(PEOPLE_TIME_INDEX);
         }
 
+        public boolean belowMemThreshold() {
```

Comment:
9cffc4c c07e210 1f5a72c 0612317

sorry... they are in different branches dues to the weird failure of the test...